docs/development/l10n.rst

```diff
@@ -179,6 +186,36 @@ translated HTML in Portuguese would be::
 
   <img src="{icon}" alt="ícone do escudo">
 
+Attribute ``title``
+"""""""""""""""""""
+
+Links (``<a>``) and abbreviations (``<abbr>``) sometimes rely on
+an additional ``title`` attribute. The content of that attribute is
+usually shown when placing a cursor over the link or abbreviation.
+::
+
+  <a id="recommend-tor" title="How to install Tor Browser" href="{url}">Learn how to install it</a>
+
+It is necessary to translate the contents of any ``title`` attribute.
+The correctly translated HTML in Spanish would be::
+
+  <a id="recommend-tor" title="Cómo instalar Tor Browser" href="{url}">Aprenda cómo instalarlo</a>
+
+As explained above, the text ``recommend-tor`` of the ``id``
+attribute in the ``<a>`` element should not be translated. Neither
+should the ``{url}`` placeholder of ``href`` attribute. Only the text
+of the ``title`` attribute (``"How to install Tor Browser"``)
+should be translated.
+
+Other attributes
+""""""""""""""""
+
+No attribute other than ``alt`` and ``title`` should be translated.
+
+In particular, please make sure the attributes ``class``, ``id``,
+``height``, ``href``, ``rel``, ``src`` and ``width``
+are never translated.
```

Comment:
I defer to @rmol here, but it might be worth qualifying this a little bit for `href` -- there may be situations where linking to a different URL in the target language would be appropriate. Perhaps simply asking translators to comment in Weblate when in doubt is the way to go here.

Reply:
Very good point. That will also be highly dependent on the URL.

_Thinking aloud: in a case like this, could the localized URLs be given to the translators via string context?_

Reply:
Haven't had coffee yet, but I'm thinking if a URL were to vary like that, it would probably be done via a `{url}` placeholder, so the translators wouldn't have to worry about it. I'd keep `href` in the list of things not to touch.